test/org/sagebionetworks/bridge/dynamodb/DynamoUserConsent3Test.java

```diff
@@ -0,0 +1,33 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class DynamoUserConsent3Test {
```

Comment:
We have JSON annotations in the code, so I wanted to make sure we were testing them. If we don't need them, can we remove them?